bundles/org.openhab.binding.dbquery/src/main/resources/OH-INF/binding/binding.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<binding:binding id="dbquery" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:binding="https://openhab.org/schemas/binding/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/binding/v1.0.0 https://openhab.org/schemas/binding-1.0.0.xsd">
+
+	<name>DBQuery Binding</name>
+	<description>This is the binding for DBQuery.</description>
```

Comment:
Could you be a bit more verbose, so that somebody who hasn't heard about it gets a clue what this is about?